hilti/src/rt/fiber.cc

```diff
@@ -174,6 +174,9 @@ std::unique_ptr<Fiber> Fiber::create() {
 }
 
 void Fiber::destroy(std::unique_ptr<Fiber> f) {
+    if ( f->_state == State::Running || f->_state == State::Yielded )
+        f->abort();
```

Comment:
`abort` asserts that the fiber is `Yielded`, but we could also abort a `Running` fiber here.

Reply:
yeah, Running can't really happen here I think, changing to leave out